client/src/index.css

```diff
@@ -19,7 +19,35 @@
     --help-side-color: #eaf7c3;
     --input-border: #a2c0d4;
   }
-  
+
   body {
     font-family: var(--main-font);
-  }
\ No newline at end of file
+    margin: 0;
+  }
+.loading-container{
```

Comment:
These css lines shouldn't be here in the root `index.css`, they belong to another place